handlers/movieHandler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/RealImage/moviebuff-sdk-go"
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+
+	"github.com/ashwinspg/explore-golang/config"
+)
+
+//GetMovieHandler - get movie details
+func GetMovieHandler(w http.ResponseWriter, r *http.Request) {
+	UUID := chi.URLParam(r, "id")
+
+	moviebuffObj := moviebuff.New(moviebuff.Config{
+		HostURL:     config.MOVIEBUFF_URL,
+		StaticToken: config.MOVIEBUFF_TOKEN,
+	})
+
+	movieDetail, err := moviebuffObj.GetMovie(UUID)
+	if err != nil {
+		logrus.Error(err)
+	}
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
Set the httpstatus header as well. If err == nil, w.WriteHeader(http.StatusOk). If err != nil, w.WriteHeader(http.StatusInternalServerError)
Clients will read the status to determine whether or not the request is a success

Reply:
Sure, I'll change it..

Reply:
I pushed the changes as you suggested